src/Maestro/Maestro.Web/Controllers/AzDevController.cs

```diff
@@ -46,5 +46,20 @@ public async Task<IActionResult> GetBuildStatus(string account, string project,
                         Convert.ToBase64String(Encoding.UTF8.GetBytes(":" + token)));
                 });
         }
+
+        [HttpGet("getFile/{account}/{project}/{repo}/{*filePath}")]
+        public async Task<IActionResult> GetSourceFile(string account, string project, string repo, string filePath)
+        {
+            string token = await TokenProvider.GetTokenForAccount(account);
+            return await HttpContext.ProxyRequestAsync(
+                s_lazyClient.Value,
+                $"https://dev.azure.com/{account}/{project}/_apis/git/repositories/{repo}/items?path=/{filePath}&includeContent=true&resolveLfs=true&api-version=5.1",
```

Comment:
Aren't the dependencies defined in version.details.xml different on different branches?  how do you define a branch?  I still feel like this is mostly a lie or misrepresentation  since we don't do any filtering based on branch.  @mmitche ?